swaybar/status_line.c

```diff
@@ -418,15 +480,29 @@ bool handle_status_line(struct bar *bar) {
 		if (line[0] == '{') {
 			// detect i3bar json protocol
 			json_object *proto = json_tokener_parse(line);
-			json_object *version;
 			if (proto) {
+
+				json_object *version;
 				if (json_object_object_get_ex(proto, "version", &version)
 							&& json_object_get_int(version) == 1
 				) {
 					sway_log(L_DEBUG, "Switched to i3bar protocol.");
 					bar->status->protocol = I3BAR;
-					i3json_handle_data(bar, line_rest);
 				}
+
+				json_object *click_events;
+				if (json_object_object_get_ex(proto, "click_events", &click_events)
+							&& json_object_get_boolean(click_events)
+				) {
```

Comment:
Here as well

Reply:
No, no, put the parens and brace on the same line.

Reply:
Ah, ok. 
I have just made same thing as in a code right above. Condition there is a multi-line one. 
if I do it on a same line it is 116 chars long...
Can I split it like this:
if (json_object_object_get_ex(proto, "click_events", &click_events) 
      && json_object_get_boolean(click_events)) {

or just keep it 116 chars long?

Reply:
I would split it on the boolean operator and let it exceed 80 columns if necessary.